apps/trading/components/vega-wallet-connect-button/vega-wallet-connect-button.spec.tsx

```diff
@@ -27,6 +27,16 @@ describe('VegaWalletConnectButton', () => {
   it('should fire dialog when not connected', () => {
     render(generateJsx({ pubKey: null } as VegaWalletContextShape));
 
+    const button = screen.getByTestId('connect-vega-wallet');
+    expect(button).toHaveTextContent('Get started');
+    fireEvent.click(button);
+    expect(mockUpdateDialogOpen).toHaveBeenCalled();
+  });
+
+  it('should be "Connect" in the button', () => {
```

Comment:
Can you rename: 'it should render "Connect" when browser wallet is detected'

Reply:
done